www/layouts/collections/single.html

```diff
@@ -47,7 +47,7 @@ <h1>
             {{- $course_list_data = $course_list_data | append (dict $id $data) -}}
           {{ end }}
         {{ else }}
-          {{ $errorMessage := printf "Failed to fetch collections through %v on %v" $url site.BaseURL }}
+        {{ $errorMessage := printf "Failed to fetch course data from %v. Skipping course on page %v" $url site.BaseURL }}
```

Comment:
```suggestion
          {{ $errorMessage := printf "Failed to fetch course data from %v. Skipping %v on %v" $url $id $collection }}
```

OK, I didn't naturally trigger the error message, but I was able to trigger it by changing `resources.GetRemote $url` to `resources.GetRemote "https://mit.edu/some/bogus/path/so/requests/404"`.

I don't think the error message is accurate as written. Right now, it will say things like:
> Failed to fetch course data from https://live-qa.ocw.mit.edu/courses/2-57-nano-to-macro-transport-processes-spring-2012/data.json. Skipping course on page http://ocw.mit.edu
> Failed to fetch course data from https://live-qa.ocw.mit.edu/courses/2-627-fundamentals-of-photovoltaics-fall-2013/data.json. Skipping course on page http://ocw.mit.edu

**Issue:** "On page http://ocw.mit.edu" isn't really accurate. It's on that site, but the error message does not currently indicate what page was affected.

With the suggestion above, the error messages will look like:
I suggest changing to:
> Failed to fetch course data from https://live-qa.ocw.mit.edu/courses/2-627-fundamentals-of-photovoltaics-fall-2013/data.json. Skipping courses/2-627-fundamentals-of-photovoltaics-fall-2013 on Page(/collections/energy.md)
> Failed to fetch course data from https://live-qa.ocw.mit.edu/courses/22-312-engineering-of-nuclear-reactors-fall-2015/data.json. Skipping courses/22-312-engineering-of-nuclear-reactors-fall-2015 on Page(/collections/energy.md)